.github/scripts/build-image.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# build and push docker image
+#
+
+set -o errexit
+set -o pipefail
+
+HELM_VERSIONS="$(grep 'ENV HELM2_VERSION' Dockerfile | sed -e 's/ENV HELM2_VERSION v//') $(grep 'ENV HELM_VERSION' Dockerfile | sed -e 's/ENV HELM_VERSION v//')"
+GCLOUD_SDK_VERSION="$(grep google/cloud-sdk Dockerfile | sed -e 's#FROM\ google/cloud-sdk:##'  -e 's#-alpine##')"
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+IS_MASTER="false"
+if [ "${GITHUB_REF}" == 'refs/heads/master' ]; then  IS_MASTER=true; fi
+
+echo "DEBUG github environment: "
+set | grep GITHUB
+
+for HELM in ${HELM_VERSIONS}; do
```

Comment:
Same story here. Use [matrix strategy](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix), (existing) environment variable for the install script and a corresponding `Dockerfile` build argument to pass matrix parameter into the image build. Then use some [docker actions](https://github.com/marketplace/actions/build-and-push-docker-images) to actually build and push the images.